android/envoy/src/main/kotlin/org/greatfire/envoy/CronetNetworking.kt

```diff
@@ -47,12 +51,22 @@ object CronetNetworking {
                 .setStoragePath(cacheDir.absolutePath)
                 .enableHttpCache(CronetEngine.Builder.HTTP_CACHE_DISK, cacheSize * 1024 * 1024)
         }
-        if (!envoyUrl.isNullOrEmpty()) {
-            builder = builder.setEnvoyUrl(envoyUrl)
+
+        envoyUrl?.let {
+            Log.e(TAG, "envoyUrl is unsupported here now!")
+            // builder = builder.setEnvoyUrl(it)
+        }
```

Comment:
should the parameter be removed instead?

Reply:
Yeah, this is something I improved in the masque branch... I ended up fixing up a lot of the cronet w/ proxy stuff there, which is part of why I want to merge this one :)